### ISSUE TITLE
Error: Cannot find module 'dom-serializer'

### ISSUE THREAD

**Issue by neemah** [html-to-text/node-html-to-text#269]
**Observed output**

```
Error: Cannot find module 'dom-serializer'
Require stack:
- /...../project/node_modules/html-to-text/lib/html-to-text.cjs
```

**Expected output**

```
No errors about dom-serializer
```

**Version information**

- html-to-text: latest
- node: 16
----

I did not find dom-serializer in package.json of html-to-text package, it seems this is the reason behind error. Added this librabry to our project solved and issue.

**Comment by KillyMXI** [comment]
Hmm.
Weird that you got the error.
The dependency chain is `html-to-text -> htmlparser2 -> domutils -> dom-serializer`
So it must be present in `node_modules`.

But anyway, good point, looks like I've added it explicitly in `html-to-md/package.json` and forgot in `html-to-text/package.json`

Thanks

**Comment by KillyMXI** [comment]
Published version 9.0.3 with the fix.
